lib/core/sync_job_runner.rb

```diff
@@ -43,8 +43,8 @@ def execute
     private
 
     def validate_configuration!
-      expected_fields = @connector_instance.configurable_fields.keys
-      actual_fields = @connector_settings.configuration.keys
+      expected_fields = @connector_instance.configurable_fields.keys.map(&:to_s)
```

Comment:
Could we maybe do the `.with_indifferent_access` conversion in the `@connector_instance.configurable_fields` base method? Or add the `.with_indifferent_access` here?

Reply:
Also added .with_indifferent_access in https://github.com/elastic/connectors/pull/166/commits/f7b54acca15a595920aae7efafd133d03bb05e9f